pkg/certificate/certificate.go

```diff
@@ -35,3 +47,15 @@ func (c *Certificate) GetPrivateKey() pem.PrivateKey {
 func (c *Certificate) GetIssuingCA() pem.RootCertificate {
 	return c.IssuingCA
 }
+
+// ShouldRotate determines whether a certificate should be rotated.
+func (c *Certificate) ShouldRotate() bool {
+	// The certificate is going to expire at a timestamp T
+	// We want to renew earlier. How much earlier is defined in renewBeforeCertExpires.
+	// We add a few seconds noise to the early renew period so that certificates that may have been
+	// created at the same time are not renewed at the exact same time.
+
+	intNoise := rand.Intn(maxNoiseSeconds-minNoiseSeconds) + minNoiseSeconds /* #nosec G404 */
```

Comment:
For style consistency
```suggestion
	intNoise := rand.Intn(maxNoiseSeconds-minNoiseSeconds) + minNoiseSeconds // #nosec G404 
```

Reply:
done!